### PR TITLE
Add breaking changes and release highlights

### PR DESCRIPTION
**Pull request by dedemorton**
Changes:

* Adds breaking changes based on the list from the relnotes pruned down to include only BCs that might impact users.
* Adds release highlights taken and pruned from the release blog. Other teams are planning to do the same (provide a short list and point to the blog for details).
* Minor cleanup to remove extraneous comments and make the section intros more consistent with other doc sets.

Please respond to questions flagged as comments for reviewers.

### CODE REVIEW

**Comment by dedemorton** [comment]
FYI @lcawl

**Comment by mikemadden42** [comment]
jenkins retest this please

File: libbeat/docs/breaking-7.4.asciidoc [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+//Installation and Upgrade Guide
+
+//tag::notable-breaking-changes[]
+
+[float]
+==== Golang version update
+
+The Beats platform has been updated to use Golang 1.12.7.
+
+// REVIEWERS: Is the above statement correct (taken from the relnotes)? The
```
Comment (dedemorton) [inline libbeat/docs/breaking-7.4.asciidoc]: @mikemadden42 Thanks for the review! I do still have an open question, though. Is this statement correct? If so, why does the golang version shown here conflict with the version info in version.asciidoc?

File: libbeat/docs/breaking-7.4.asciidoc [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+[options="header"]
+|====
+|Old field                 |New field
+|`log.original`            |`event.original`
+|`cisco.asa.list_id`       |`cisco.asa.rule_name`
+|====
+
+
+//REVIEWERS: Do we want to mention the functionbeat deployment/pkg changes here?
```
Comment (dedemorton) [inline libbeat/docs/breaking-7.4.asciidoc]: @mikemadden42 Here's another question. I'm assuming the answer is "no" based on an email thread I saw go by.

File: libbeat/docs/breaking-7.4.asciidoc (hunk 1 above)
Comment (dedemorton) [reply libbeat/docs/breaking-7.4.asciidoc]: @mikemadden42 I'm using the version that's shown in version.asciidoc (1.12.9). Let me know if that's not correct.

File: libbeat/docs/breaking-7.4.asciidoc (hunk 2 above)
Comment (dedemorton) [reply libbeat/docs/breaking-7.4.asciidoc]: @mikemadden42 I'm going to go ahead and merge this PR (as soon as the CI is built) so the content will be available tomorrow. We can still make changes/additions if necessary. Just let me know.

**Pull request merged**